vizro-core/src/vizro/actions/_actions_utils.py

```diff
@@ -66,34 +63,97 @@ def _apply_filters(
     return data_frame
 
 
-def _apply_filter_interaction(
-    data_frame: pd.DataFrame,
-    ctds_filter_interaction: List[CallbackTriggerDict],
-    target: str,
+def _apply_chart_filter_interaction(
```

Comment:
```suggestion
def _apply_graph_filter_interaction(
```
and in subsequent places